syntax = "proto2";

package dempy;

import "sample.proto";
//import "annotation."

message SampleList {
  repeated TimeseriesSample timeseries = 1;
  repeated ImageSample images = 2;
  repeated VideoSample videos = 3;
}

//message AnnotationList {
//  repeated Annotation annotations = 1;
//}